r_scripts/data_wrangle/2013_oregon_timestratified_casecross.R

```diff
@@ -0,0 +1,201 @@
+# ------------------------------------------------------------------------------
+# Title: Oregon Case-crossover dataframe creation (only patients with single obs)
+#        and joining with and zip-level PM2.5 values. 
+# Author: Jingyang Liu
+# Date Created: Nov 27 2017     
+# R version: 3.3.3
+# ------------------------------------------------------------------------------
+
+# Notes ----
+# Notes 8/16/16: I updated smoke data with population weighted averages for zip
+# This code also assumed only one visit, so I remove the people with multiple
+# admissions for the outcome
+
+# Notes 10/3/16: I'm retaining the county assignment of the observation so I
+# can join the county population weighted averages as well for comparison
+# and to analyze morbidity similar to Rish
+
+# Libraries ----
+# load libraries
+#library(tidyverse) # tidyverse not on the server
+# if tidyverse is not available, use dplyr and readr
+library(dplyr) 
+library(readr)
+library(lubridate) # working with date
+# parallel computing libraries
+# library(foreach) 
+# library(doParallel)
+
+# Read in smoke data created in loop -------------------------------------------
+
+# read in zipcode level populatoin-weighted pm
+read_path5 <- paste0('../data/smoke/zip_pm_to_merge_with_acap.csv')
+zip_smoke <- read_csv(read_path5) # 63801 rows
+
+# descriptives of the two smoke datasets
+summary(zip_smoke)
+
+# Zipcode PM2.5 estimates
+# create lag variables that take smoke values from n previous days for zipcodes
+zip_smoke_w_lag <- zip_smoke %>% arrange(ZIPCODE, date) %>%
```

Comment:
I want creation of zip smoke lags in a separate script. Ask me later when we have time.